docs/data/date-pickers/custom-field/custom-field.md

```diff
@@ -158,10 +158,17 @@ you can replace the field with an `Autocomplete` listing those dates:
 
 {{"demo": "PickerWithAutocompleteField.js", "defaultCodeOpen": false}}
 
+### Using a read-only `TextField`
+
+If you only want to allow the user to pick a value through the views,
```

Comment:
Maybe too nitpicking, but the word "only" in this position gives me a feeling more of being related to the developer's personal desire, rather than describing and enforcing a functional behavior.
Maybe omiting it and reinforcing the date selection through the view could emphasize more the restriction ? Something like:

"If you want to limit users to select a value only through the views"
or
"If you want users to select a value exclusively through the views"

Reply:
I went with the 2nd one :pray: